README.md

```diff
@@ -135,8 +135,7 @@ cairo = library_dependency("cairo", aliases = ["libcairo-2", "libcairo"], depend
     e.g. whether or not has the correct version. To do so, pass a function that takes 
     (name,handle) as an argument and returns `true` if the library is usable and `false` 
     it not. The `name` argument is either an absolute path or the library name if it is a
-    global system library, while the handle is a handle that may be passed to `ccall` or
-    `dlsym` to check library symbols or the return value of function. Note however that it 
+    global system library, while the handle is a handle that may be passed to  `dlsym` to check library symbols or the return value of function. Note however that it 
```

Comment:
I found it in contributing, but rather read it as: for code. But i don't see it in style guide section?